python/tidy/servo_tidy/tidy.py

```diff
@@ -694,6 +698,46 @@ def check_spec(file_name, lines):
                 brace_count -= 1
 
 
+def check_config_file(config_file, print_text=True):
+    if conf_file and config_file == CONFIG_FILE_PATH:
+        lines = conf_file.splitlines(True)
+    else:
+        # Needed for running tests
+        with open(config_file, "r") as content:
+            lines = content.read().splitlines(True)
```

Comment:
It doesn't have to be this way, since parsing (and validating the config file) should be the first step, and only then we should get the configuration required for tidy.